xml/System.Runtime.InteropServices/AnsiStringMarshaller.xml

```diff
@@ -70,10 +70,21 @@
         <Parameter Name="buffer" Type="System.Span&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="str">To be added.</param>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="str">The string to marshal.</param>
+        <param name="buffer">Buffer that may be used for marshalling.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Runtime.InteropServices.AnsiStringMarshaller" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The `buffer` must not be movable - that is, it should not be
+            on the managed heap or it should be pinned.
+
+          ]]></format>
+        </remarks>
+        <seealso cref="F:System.Runtime.InteropServices.CustomTypeMarshallerFeatures.CallerAllocatedBuffer" />
+        <seealso cref="F:System.Runtime.InteropServices.CustomTypeMarshallerFeatures.CallerAllocatedBuffer" />
```

Comment:
GitHub is making a mess. Telling me the suggestions cannot be applied because the diff is outdated. I had to submit a batch of updates via command line, but now it's showing some changes twice. Sigh...

```suggestion
```